wordpress-mobile/WordPress-FluxC-Android
# Jetpack CP: Add Jetpack-tunneled plugins-related functionalities.

## PR Description

This PR is adding functionalities to:
-  Install a plugin,
- Fetch a plugin's information
  
to .org sites that are connected using Jetpack Connection Package.

## To Test

1. Prepare a site connected with Jetpack CP. An easy way is to start a .org site, don't install Jetpack, and instead install Jetpack Backup plugin. Open the plugin's setting and connect it to your wpcom account.
2. Open Example app, then sign in to your wpcom account.
3. Go to "Plugins", then select the site created on step 1.
4. Use the "Install and activate a plugin" button and try installing a plugin. You will need to enter a plugin's slug, which is the `http://wordpress.org/support/plugin/<SLUG>` part on the Plugin Directory. Observe the log and ensure it says the plugin is installed and activated.
5. Use the "Fetch a plugin from site" button. Try fetching for `jetpack-backup/jetpack-backup`, and observe the log. It will show the plugin's name and description.

Extra: you can also use Flipper to observe the network calls on both buttons's action and check the request/response there.
